bcgov/HOUS-permit-portal
# Feature/bphh 883/jurisdiction list

## Description

Add a jurisdiction page available everywhere, even when logged out.
Add regional district name to jurisdiction search data.
add regional district and address to jurisdiction creation.
Fix last_signed_in_at search field for users
Fix name clash for jurisdiction search concern methods (order, query, etc)
Add locality type filter to jurisdiction select search

## What type of PR is this? (check all applicable)

- [ x] 🍕 Feature

## Related Tickets & Documents

https://hous-bssb.atlassian.net/browse/BPHH-883
https://hous-bssb.atlassian.net/browse/BPHH-969

## Steps to QA

log in as super admin, create a new jurisdiction. provide a regional district (like nanaimo)
log out. Navigate to new jurisdiciton index through menu. See the regional district and name.
